src/libs/Navigation/AppNavigator/MainDrawerNavigator.js

```diff
@@ -81,5 +85,8 @@ export default withOnyx({
     betas: {
         key: ONYXKEYS.BETAS,
     },
+    isLoadingBetas: {
+        key: ONYXKEYS.IS_LOADING_BETAS,
+    },
```

Comment:
Could we get a quick one or two sentence explanation about why this is needed?

Reply:
As a comment in the code or just as a comment here?

The "Select a free workspace" transition shows a blank sidebar and main content panel if the user was not signed in to NewDot before transitioning. If the betas are loading we show a loading spinner while the betas are loading so that no blank panels are shown. However, this is admittedly not a perfect fix because sometimes the panels are still blank. I'm not sure exactly what needs to be loaded before the side panel and the main content panel are populated with data.

Reply:
I disabled the `isLoadingBetas` part of this condition:

https://github.com/Expensify/App/blob/7e6570800ac926527e818e4dd331b549667fb680/src/libs/Navigation/AppNavigator/MainDrawerNavigator.js#L54-L57

But the test case mentioned seems to work fine without this code.

@roryabraham I think we should treat this like a separate bug and if it pops up we can handle it, but maybe good to have you test this flow as well on the latest changes.

Reply:
Oh I think I see it now. Not something we should fix right now IMO. The sidebar eventually populates with data.